config/routes.rb

```diff
@@ -57,7 +57,9 @@
   end
   resources :cuisines, only: [:show] do
     # ここにindexアクションを含めて良いのか?
-    resources :favorites, only: [:create, :destroy]
+    # resources :favorites, only: [:create, :destroy]
+    post :favorite
+    delete :favorite
```

Comment:
これでどうですかね。

```suggestion
    post :favorite, to: :create, controller: 'favorites'
    delete :favorite, to: :destroy, controller: 'favorites'
```

Reply:
下記のエラーが表示されるようになりました...
```
AbstractController::ActionNotFound - The action 'favorite' could not be found for FavoritesController:
```

Reply:
失礼、toだとダメっぽいですね。actionだとどうですかね？

```suggestion
    post :favorite, action: :create, controller: 'favorites'
    delete :favorite, action: :destroy, controller: 'favorites'
```

https://railsguides.jp/routing.html#%E5%8D%98%E6%95%B0%E5%BD%A2%E3%83%AA%E3%82%BD%E3%83%BC%E3%82%B9